pandas/core/strings.py

```diff
@@ -295,20 +295,24 @@ def str_count(arr, pat, flags=0):
 
 def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
     """
-    Return boolean Series/``array`` whether given pattern/regex is
-    contained in each string in the Series/Index.
+    Test if pattern or regex is contained within each string of a Series.
```

Comment:
This method is used by `Index` as well which was removed here - please be sure to add that back